tests/api/test_bugzilla.py

```diff
@@ -0,0 +1,161 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+import json
+from datetime import datetime
+
+from purview.models.user import User
+from purview.models.bugzilla import BugzillaBug
+from purview.models.distgit import DistGitCommit
+from purview.models.errata import Advisory
+
+
+def test_get_bug(client):
+    """Test getting a resource from Neo4j with its relationships."""
+    tbrady = User.get_or_create({
```

Comment:
Should we have general functions like 'create_user' and 'create bug' so that we can reduce duplication?

Reply:
I was thinking so but it doesn't save much code.

Reply:
cool